### DISCUSSION TITLE
What are the terms of use for the ESPN API?

### DISCUSSION THREAD

**Discussion by znmeb** [saiemgilani/wehoop#30] [Q&A]
I'm about to do some detailed analysis that will require updated WNBA player box score data daily. I've been buying WNBA data but if `wehoop` can supply what I need I won't need to buy data any more.

What are the terms of use of the API? What may I do and what must I not do in order to not incur anyone's wrath / cease and desist letter?

**Comment by saiemgilani** [comment]
Should be updated nightly, yes. For now, at minimum, I can guarantee it'll be updated by the next morning from ESPN. But soon we'll have more data from WNBA.com available for loading which will be at minimum better.

Is the next morning bit acceptable?

Oh and if you're willing, it's definitely possible to get live boxscores too from the package from ESPN, if not WNBA proper

**Comment by znmeb** [comment]
Next morning is fine with me. I'm more concerned with 

a) how often I can hit the service(s) without getting my hand slapped / IP address blocked, etc., and
b) whether I have to keep my results private or whether I can publish analysis. You know, the stuff ESPN / NCAA / WNBA lawyers and accountants care about.

Scrapers tend to upset service providers, and sports data providers tend to be more easily upset. I've had my IP address blocked, and some places are fine with you accessing their data *manually* as long as you attribute them in writing. But a daily automated process and possibly a womens' basketball analytics blog is quite another story.
